scripts/check_changelog_fragments.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# This script is intended to run during CI, however it can be run locally by
+# committing changelog fragments before executing the script. If the script
+# finds an issue with your changelog fragment, you can un-stage the fragment
+# from being committed and fix the issue.
+
+CHANGELOG_DIR="changelog.d"
+FRAGMENT_TYPES="breaking|security|deprecation|feature|enhancement|fix"
```

Comment:
It might be worth a note here and in `generate-release-cue.rb` that these lists have to remain in sync with the `README.md` file.